tools/Config.mk

```diff
@@ -48,7 +48,7 @@ else ifeq ($(V),)
 endif
 
 ifeq ($(ECHO_BEGIN),)
-  export ECHO_BEGIN=@echo
+  export ECHO_BEGIN=@echo 
```

Comment:
Can we add a comment here to prevent others from mis-removing it again if we choose to add trailing space here?
```
export ECHO_BEGIN=@echo # keep a trailing space here
```

Reply:
Let's see if that comment will not bring new problems 🙂

Reply:
Done.